specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -3601,6 +3609,283 @@
+            ]
+          },
+          "description": "The disk type used by virtual machine in hostpool session host."
+        },
+        "vmCustomConfigurationUri": {
+          "type": "string",
+          "description": "The uri to the storage blob containing scripts to be run on the virtual machine after provisioning."
+        },
+        "imageInfo": {
+          "$ref": "#/definitions/ImageInfoProperties",
+          "description": "Image configurations of HostPool."
+        },
+        "domainInfo": {
+          "$ref": "#/definitions/DomainInfoProperties",
+          "description": "Domain configurations of HostPool."
+        }
+      }
+    },
+    "ImageInfoProperties": {
+      "description": "Image configurations of session host in a HostPool.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Gallery",
+            "StorageBlob",
+            "CustomImage"
+          ],
+          "x-ms-enum": {
+            "name": "ImageType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Gallery",
+                "description": "Using default gallery images offered by azure market place."
+              },
+              {
+                "value": "StorageBlob",
+                "description": "Using a VHD stored in a storage blob."
+              },
+              {
+                "value": "CustomImage",
+                "description": "Using custom image or custom shared image."
+              }
+            ]
+          },
+          "type": "string",
+          "description": "The type of image session hosts use in the hostpool.",
+          "x-nullable": false
+        },
+        "marketPlaceInfo": {
+          "$ref": "#/definitions/MarketPlaceInfoProperties",
+          "description": "The values to uniquely identify a gallery image."
+        },
+        "storageBlobUri": {
+          "type": "string",
+          "description": "The uri to the storage blob which contains the VHD. Image type must be StorageBlob."
+        },
+        "customId": {
+          "type": "string",
+          "description": "The resource id of the custom image or shared image. Image type must be custom."
+        }
+      }
+    },
+    "MarketPlaceInfoProperties": {
+      "description": "Image configurations of HostPool.",
+      "type": "object",
+      "properties": {
+        "offer": {
+          "type": "string",
+          "description": "The offer of the image."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "The publisher of the image."
+        },
+        "sku": {
+          "type": "string",
+          "description": "The sku of the image."
+        },
+        "exactVersion": {
+          "type": "string",
+          "description": "The exact version of the image."
+        }
+      }
+    },
+    "DomainInfoProperties": {
+      "description": "Domain configurations of HostPool.",
```

Comment:
should this be the "Domain configuration of session hosts."?

Reply:
ok